scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,116 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import json
+import urllib2
+import logging
+import datetime
+import dateutil
+import operator
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
+
+logger = logging.getLogger(__name__)
+
+
+def retrieve_data(url):
+    """ Fetch data and decode json """
+    response = urllib2.urlopen(url)
+    data = json.load(response)
+    return data
+
+
+def get_popular_nodes():
+    """ Fetch data from url that returns dict with a list of popular nodes from piwik """
+    # TODO Shouldn't hardcode URL - only works locally
+    discover_url = 'http://127.0.0.1:5000/api/v1/explore/activity/popular/raw/'
+    return retrieve_data(discover_url)
```

Comment:
We have [`requests`](http://docs.python-requests.org/en/v2.5.3/) in our `requirements.txt`. Let's use that API instead of wrapping old-skool `urllib2` API.